migration/migrator/migrations/course/20210621125502_thread_announce.py

```diff
@@ -0,0 +1,33 @@
+"""Migration for a given Submitty course database."""
+
+
+def up(config, database, semester, course):
+    """
+    Run up migration.
+
+    :param config: Object holding configuration details about Submitty
+    :type config: migrator.config.Config
+    :param database: Object for interacting with given database for environment
+    :type database: migrator.db.Database
+    :param semester: Semester of the course being migrated
+    :type semester: str
+    :param course: Code of course being migrated
+    :type course: str
+    """
+    database.execute("ALTER TABLE threads ADD COLUMN IF NOT EXISTS announced timestamp(6) with time zone DEFAULT NULL")
+
+
+def down(config, database, semester, course):
+    """
+    Run down migration (rollback).
+
+    :param config: Object holding configuration details about Submitty
+    :type config: migrator.config.Config
+    :param database: Object for interacting with given database for environment
+    :type database: migrator.db.Database
+    :param semester: Semester of the course being migrated
+    :type semester: str
+    :param course: Code of course being migrated
+    :type course: str
+    """
+    database.execute("ALTER TABLE threads DROP announced")
```

Comment:
Typically we don't want a down migration to delete data when we can still run Submitty with that there. If for some reason Submitty needs to be rolled back it would be handy to not need to delete all that data. The down migration for this could just be pass since your up migrations does a check to make sure it doesn't already exist.